docs/data/tree-view/rich-tree-view/focus/focus.md

```diff
@@ -28,9 +25,22 @@ return <RichTreeView apiRef={apiRef} items={ITEMS}>;
 `apiRef` will be undefined during the first render and will then contain methods allowing you to imperatively interact with the Tree View.
 :::
 
+### Focus a specific item
+
+Use the `apiRef.focusItem` method to focus a specific item.
```

Comment:
above we are using "Use the `setItemExpansion` API method to ..." and here it is different.
Should we consolidate how we write this?
```suggestion
Use the `focusItem` API method to focus a specific item.
```
🤷🏼